tests/functional/workloads/cnv/test_cnv_mon_osd_failure.py

```
@@ -60,89 +61,105 @@ def test_mon_failures(self, mon_count, request, setup_cnv_workload):
 
         """
         ceph_obj = CephCluster()
-        logger.info(f"testing mon failures scenario with {mon_count} mon")
+        logger.test_step(f"Simulate {mon_count} mon failure(s)")
 
         mons = ceph_obj.get_mons_from_cluster()[:mon_count]
 
         def teardown():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `teardown`

Add return type annotation: `None`

(ANN202)